### ISSUE TITLE
Is it possible to disallow "Anonymous" accounts?

### ISSUE THREAD

**Issue by Loreno10** [Erudika/scoold#235]
Users are able to make their profiles "anonymous" - is it possible to disable that?

**Comment by albogdano** [comment]
https://github.com/Erudika/scoold#anonymous-posts

**Comment by Loreno10** [comment]
Right, I somehow missed that. However, after enabling that, and restarting the whole stack, the user is still able to enable "Anonymous" access:

![image](https://user-images.githubusercontent.com/10273406/113271692-143d8000-92db-11eb-9182-515ba3c68c8e.png)

Here's the setting that I added to `scoold-application.conf`:

```ini
# Disable anonymous users
para.profile_anonimity_enabled = false
```

**Comment by Loreno10** [comment]
I think this setting is not "PRO", right?

**Comment by albogdano** [comment]
Fixed in 7cdf8bf
